### DISCUSSION TITLE
请问能否通过《关于研究生学位论文撰写规范自查的通知》中提到的格式检查？

### DISCUSSION THREAD

**Discussion by joelongLin** [sikouhjw/gdutthesis#44] [General]
公告链接：https://yjs.gdut.edu.cn/info/1040/3858.htm
研究生可查看的链接：https://2873.lun51.com/yjs
个人版：https://www.lun51.com/

请问是否能帮忙测一下？

**Comment by joelongLin** [comment]
进入之后大概能看到这样的界面~

![1647264834(1)](https://user-images.githubusercontent.com/29549049/158182646-5de40818-7cce-41bd-8d24-71f99894c431.png)

**Comment by SiriusFang2020** [comment]
把latex输出的PDF压缩成Zip之后，再上传可以实现检测，目前本人测试后主要提示的格式问题是英文字体问题

**Reply by sikouhjw** [reply]
具体提示信息是什么？能详细说说吗。

**Reply by SiriusFang2020** [reply]
提示信息如下：
![image](https://user-images.githubusercontent.com/59632817/158188159-b72d3e21-0e8b-414c-a53f-a2a0d453a858.png)
![image](https://user-images.githubusercontent.com/59632817/158188193-2a20b2c2-1775-4355-bd6f-6f125f39ba5b.png)
![image](https://user-images.githubusercontent.com/59632817/158188463-91b649ec-6a94-4147-906a-4ffa321f887e.png)

**Reply by SiriusFang2020** [reply]
我的字体设置是这样的：
![image](https://user-images.githubusercontent.com/59632817/158188789-18a815a7-67e4-4ac1-acb4-1b3677af7ae5.png)

**Reply by sikouhjw** [reply]
感谢回复，用真正的论文去测试，测试出来的数据更全面。

我目前认为这个格式提醒不重要，也就不打算改动模板了。

如果有人因为这个被某一阶段拦下了，我再给出解决方案吧。

**Reply by iTomxy** [reply]
记下一种把数字、英文无脑全部改成 Times New Romans 的方法：
1. *main.tex* 中将 [\gdutsetup -> style -> font](https://github.com/sikouhjw/gdutthesis/blob/main/main.tex#L33) 设成 `times*`
2. *gdutthesis.cls* 在 [`\cs_new_protected:cpn { __gdut_load_font_ times* : }` -> `\__gdut_setsansfont:nn`](https://github.com/sikouhjw/gdutthesis/blob/main/gdutthesis.cls#L657) 将 `Arial` 换成 `Times~ New~ Roman`

**Reply by Quantum-Electrodynamics** [reply]
它还是有几处要Arial字体的，目前我看到英文封面标题、名字啥的是Arial

**Reply by sikouhjw** [reply]
> 它还是有几处要Arial字体的，目前我看到英文封面标题、名字啥的是Arial

所以这就是矛盾所在，凭什么用无衬线中文（黑体）搭配衬线英文（TNR），这是不合理的。

**Comment by sikouhjw** [comment]
我上传了之前给研究生院老师拿来测试查重的文件，主要格式提示如下：
1. Abstract章标题-提醒-整段英文、数字和半角标点要求使用字体黑体, 实际：Arial。
2. 11-1-英文目录章标题-提醒-整段英文、数字和半角标点要求使用字体黑体, 实际：Arial。
3. 11-8-一级英文目录项ABSTR-提醒-整段英文、数字和半角标点要求使用字体Times New Roman, 实际：Arial。
4. 13-1-正文章标题-错误-目录中未找到标题，请检测标题大纲级别是否为1级，或更新目录内容保持与正文章节一致
5. 17-1-结论章标题-错误-目录中未找到标题，请检测标题大纲级别是否为1级，或更新目录内容保持与正文章节一致

我的解释如下：
1. chapter 要求是黑体，即无衬线体，`黑体` + `Arial` 才是正确的字体搭配，因此英文文字用中文字体才是错误的；
2. 目录中的 chapter，根据《撰写规范》[^1]的要求为『目录中各章题序及标题用小四号黑体，其余用小四号宋体。』，因此是《广东工业大学研究生学位论文撰写规范格式模板》[^2]上面的目录有问题，导致检测系统认为有错。

这个检测系统我认为其建立在错误的依据之上，格式提醒没有意义，那么只需要关注其『万字差错率』功能即可。

[^1]: https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073
[^2]: https://yjs.gdut.edu.cn/guangdonggongyedaxueyanjiushengxueweilunwenzhuanxieguifangeshimoban.docx

**Reply by sikouhjw** [reply]
补充：https://github.com/sikouhjw/gdutthesis-Word/issues/28#issuecomment-2095621596

**Reply by FokWaijyun** [reply]
已经和这个格式检查系统的客服咨询过，得到下面的回复：“提醒如果没有问题可以忽略，系统也不会当作错误进行处理，也不计入错误数。”
目前出现的提醒问题，主要就是正文或者标题里面的英文字体和阿拉伯数字字体，不是新罗马字体：
”提醒：此处英文、数字和半角标点要求使用字体Times New Roman, 实际：XITS-Regular。,连续6 字符“

遇到这个问题不需要修改模板，查过资料，这个XITS字体也是属于新罗马字体，这个提醒并非报错，不会影响格式检查的通过与否

**Comment by jiayouhaoba** [comment]
为什么图片都会页面置顶，感觉有点文不对图了
